Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -2361,7 +2361,11 @@ internal async static Task PresignedGetObject_Test1(MinioClient minio)
                                                 objectName,
                                                 filestream, filestream.Length, null);
                 ObjectStat stats = await minio.StatObjectAsync(bucketName, objectName);
-                string presigned_url = await minio.PresignedGetObjectAsync(bucketName, objectName, expiresInt);
+                PresingedGetObjectArgs preArgs = new PresingedGetObjectArgs()
```

Comment:
typo: PresignedGetObjectArgs has a typo in other places too
```suggestion
                PresignedGetObjectArgs preArgs = new PresignedGetObjectArgs()
```